kimchi/src/circuits/lookup/index.rs

```diff
@@ -150,7 +150,7 @@ impl<T> LookupSelectors<T> {
             xor: xor.map(f),
             chacha_final: chacha_final.map(f),
             lookup_gate: lookup_gate.map(f),
-            range_check_gate: range_check_gate.map(f),
+            range_check: range_check.map(f),
```

Comment:
Yeah, definitely revert all these changes.  There's no reason to make them and doesn't have anything to do with this PR.